api/client/lib.py

```diff
@@ -855,7 +855,7 @@ def get_geojson(access_token, api_host, region_id):
 
 
 def get_descendant_regions(access_token, api_host, region_id,
-                           descendant_level):
+                           descendant_level, include_historical):
```

Comment:
Need to include a default `include_historical=True` here as well to make it optional.

We encourage using the Client classes, in which case there's a default already, but some folks are using the lib directly and this constitutes a breaking change since it adds a new required param.

Reply:
good catch! done